.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please use linter config for html_css+js from this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Reply:
Hi, I used  linter config for html_css+js but my GitHub isn't showing the checks for the javascript

Reply:
It has been resolved